main.go

```diff
@@ -29,22 +37,22 @@ func main() {
 
 	processors, err := StartProcessors(ctx, appConfig, errChan)
 	if err != nil {
-		log.Println(errors.Wrap(err, "Error starting processors"))
+		logger.Logger.Errorw("Error starting processors", "error", err)
 		shutdown(ctx, cancel, processors)
 	}
 	// Indicate readiness
 	err = readiness.Ready(ctx, appConfig.ReadinessFilePath)
 	if err != nil {
-		log.Println(errors.Wrap(err, "Error indicating readiness"))
+		logger.Logger.Errorw("Error indicating readiness", "error", err)
 		shutdown(ctx, cancel, processors)
 	}
 
 	// Block until we receive OS shutdown signal or error
 	select {
 	case sig := <-signals:
-		log.Printf("OS Signal Received: %s", sig.String())
+		logger.Logger.Errorw("OS Signal Received", "signal", sig.String())
```

Comment:
this feels more like INFO not an ERROR as receiving the shutdown signal is expected